app/src/layouts/appLayout/appHeader/appHeader.jsx

```diff
@@ -50,14 +53,22 @@ export class AppHeader extends Component {
   };
 
   render() {
-    const { sideMenuOpened, toggleSideMenu, activeProject, assignedProjects } = this.props;
+    const {
+      sideMenuOpened,
+      toggleSideMenu,
+      activeProject,
```

Comment:
I think yes. We use activeProjectName for UI part. Because, maybe if projectKey will be smth like this : projectKEY:213dasSfF%. It will display to user.